src/ui/interactivity/gestures/drag-outside.md

```diff
@@ -0,0 +1,48 @@
+---
+title: Drag outside an app
+description: How to drag from an app to another app or the operating system.
+---
+
+You will likely want to implement
+drag and drop somewhere in your app.
+
+You have a couple potential approaches
+that you can take. One directly uses
+Flutter widgets and the other uses a package
+available on pub.dev.
+
+## Draggable and DragTarget
+
+If you want to implement drag and drop within
+your application, you can use the <code>Draggable</code>
+widget. For insight into this approach, see
+the [Drag a UI element within an app][] recipe.
+
+An advantage of using <code>Draggable</code>
+and <code>DragTarget</code> is that you can supply
+Dart code to decide whether to accept a drop.
```

Comment:
**question:** Why did you use an HTML tag here? Does the Markdown not work in this case?
```suggestion
your app, you can use the `Draggable` widget.
To learn about this approach, check out the [Drag a UI element within an app][] recipe.

An advantage of using `Draggable` and `DragTarget` is that you can supply
Dart code to decide whether to accept a drop.
```